OctoAwesome/OctoAwesome.Database/OctoAwesome.Database.csproj

```diff
@@ -7,10 +7,12 @@
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <OutputPath>..\bin\Debug\</OutputPath>
+    <AllowUnsafeBlocks>False</AllowUnsafeBlocks>
```

Comment:
Why such unnecessary changes? :P

Reply:
i think it's pretty important to explicit signal to the world, that we don't live in a unsafe time and don't need unsafe blocks for out database project